docs/network-requirements.md

```diff
@@ -24,3 +25,7 @@ number from the output of `kubectl get svc kubernetes -o yaml`._
 
 [3] _Antrea automatically adds the firewall rules to allow the WireGuard packets
 (starting from v2.4), so the manual configuration on the host is not needed._
+
+[4] _The default value is 10256, but it can be overridden in the antrea-agent
```

Comment:
Also indicate that it can be completely disabled with `antreaProxy.disableServiceHealthCheckServer`

Reply:
Updated to

```
[4] _The default value is 10256, but it can be overridden in the antrea-agent
configuration `antreaProxy.serviceHealthCheckServerBindAddress`. It is used only
for external load balancer health checks. If `antreaProxy.disableServiceHealthCheckServer`
is set `true`, the health check server listening on the port will be disabled._
```